examples/atmega328p/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "mega328-test"
+version = "0.1.0"
+authors = ["Frank Villaro-Dixon <frank@villaro-dixon.eu>"]
+edition = "2021"
+license = "Public domain"
```

Comment:
Please use `MIT OR Apache-2.0` licensing.

Reply:
If that's okay, for this small and easy example I prefer the public license as IMHO it makes it less cumbersome for everyone. Of course this doesn't change the license of the library.

Reply:
I totally get your intention here (and would love to accept it this way), but I am asking for this change for practical reasons:  Introducing a second license here means that diffent parts of the repository are licensed differently and that's always a headache.  It means, for example, you couldn't simply copy code from an example in the crate's documentation into the example.

Also, public domain licensing has very unclear legal status in a lot of jurisdictions which further complicates things.

You can still license your contribution under any alternative terms everywhere else, but for inclusion in this repository, it'll make my life a lot easier to stick to the crate license :)